app/services/foreman/renderer/scope/macros/base.rb

```diff
@@ -133,19 +133,22 @@ def save_to_file(filename, content, verbatim: false)
             desc "This is useful when rendering output is a whitespace sensitive format, such as YAML."
             required :count, Integer, desc: 'The number of spaces'
             keyword :skip1, [true, false], desc: 'Skips the first line prefixing, defaults to false', default: false
+            keyword :skip_eof, [true, false], desc: 'Skips, if the line is "EOF", defaults to false', default: false
```

Comment:
Maybe we should make this keyword more generic to skip a line based on its content? In that case we could use it to skip a line with only "EOF" string as well as with potential other magic strings. Something like `skip: 'EOF'` and then
```ruby
# ...
if (line_no && skip1) || line == skip
  result << line
# ...
```
Ideally we could also support Regexp, but that might be too much...

Reply:
Thanks. nice idea. I changed it to "skip_word" but the line need to start with the word. 
Does this match your requirements? RegEx would be possible, too but maybe as it looks like I'm the only one with such a requirement, it's maybe a little bit over-engineered :-)